flutter-studio/src/io/flutter/profiler/FlutterAllMemoryData.java

```diff
@@ -43,7 +49,11 @@
     public ThreadSafeData() { }
 
     @Override
-    public List<SeriesData<Long>> getDataForXRange(Range xRange) {
+    public List<SeriesData<Long>> getDataForXRange(Range range) {
+      // NOTE: adt-ui is off on the range, need to adjust. The view is beyojd the last item in the series
```

Comment:
typo in this comment and perhaps clarify the comment a bit.
beyojd --> beyond

Reply:
Done.